lib/SPE.js

```diff
@@ -1307,7 +1304,7 @@ SPE.utils = {
         'use strict';
 
         var sourceLength = srcArray.length,
-            newArray = [ typeof srcArray[ 0 ].clone === 'function' ? srcArray[ 0 ].clone() : srcArray[ 0 ] ],
+            newArray = [ structuredClone(srcArray[0]) ],
```

Comment:
Why did you make this change?
From what I understand, that interpolateArray function can be used with an array of number, Vector2, Vector3, Vector4, Color.
structuredClone api is not aware of threejs clone method on those instances. It will destroy the Vector3 type, and make it an object:
```
let v=new THREE.Vector3(1,2,3)
Vector3 {x: 1, y: 2, z: 3}
structuredClone(v)
{x: 1, y: 2, z: 3}
```